ansible/roles/kubernetes/files/make-ca-cert.sh

```diff
@@ -94,7 +94,12 @@ octets=($(echo "${service_range}" | sed -e 's|/.*||' -e 's/\./ /g'))
 service_ip=$(echo "${octets[*]}" | sed 's/ /./g')
 
 # Determine appropriete subject alt names
-declare -a san_array=(IP:${cert_ip} IP:${service_ip} DNS:kubernetes DNS:kubernetes.default DNS:kubernetes.default.svc DNS:kubernetes.default.svc.${dns_domain} DNS:${master_name})
+declare -a san_array=(IP:${cert_ip} IP:${service_ip} DNS:kubernetes DNS:kubernetes.default DNS:kubernetes.default.svc DNS:kubernetes.default.svc.${dns_domain})
```

Comment:
Would would be "all master IP addresses"? Do you mean the internal cluster master IP or the external IPs of all masters?